packages/twenty-front/src/modules/settings/page-layout/utils/addWidgetToTab.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  type PageLayoutTab,
+  type PageLayoutWidget,
+} from '../states/savedPageLayoutsState';
+
+export const addWidgetToTab = (
```

Comment:
I like this util, introducing this kind of util could ease testing (as utils are easier to tests that hooks that are reactive)

They are other places in your code where your make this kind of construct/deconstruct logic in the hook and introducing utils will ease maintanability